apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -193,7 +180,7 @@ export default function TemplateEditorPage() {
             <UserPreference activePage={activePage} setActivePage={setActivePage} />
           </When>
           {!isLoading && !isIntegrationsLoading ? (
-            <TemplateEditor activeStep={activeStep} activePage={activePage} templateId={templateId} />
+            <TemplateEditor activeStepIndex={activeStepIndex} activePage={activePage} />
```

Comment:
What's the reason we don't need the `templateId` anymore?

Reply:
the `TemplateEditor` didn't have a props interface and `templateId` wasn't used at all...